Basics/DataInventory.ipynb

```diff
@@ -0,0 +1,298 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Exploring the Shared Datasets in the LSST Science Platform\n",
+    "<br>Owner(s): **Phil Marshall** ([@drphilmarshall](https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@drphilmarshall)), \n",
+    "<br>Last Verified to Run: **2018-08-05**\n",
+    "<br>Verified Stack Release: **16.0**\n",
+    "\n",
+    "In this notebook we'll take a look at some of the datasets available on the LSST Science Platform. \n",
+    "\n",
+    "### Learning Objectives:\n",
+    "\n",
+    "After working through this tutorial you should be able to: \n",
+    "1. Start figuring out which of the available datasets is going to be of most use to you in any given project; \n",
+    "2. Plot the patches and tracts in a given dataset on the sky;\n",
+    "3. List the available catalogs in a given dataset.\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "## Set-up\n",
+    "You can find the Stack version that this notebook is running by using eups list -s on the terminal command line:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# What version of the Stack am I using?\n",
+    "! echo $HOSTNAME\n",
+    "! eups list -s | grep lsst_distrib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "For accessing the datasets using the Butler, and then visualizing the results, we'll need the following modules:"
```

Comment:
Due to alphabetical ordering of directories, I started executing this notebook before GettingStarted/FindingDocs.ipynb which describes how to install the stackclub package. I recommend to include a note to look at that notebook or other instructions for installing stackclub.

Reply:
Oh, good catch. I think the template notebook needs a cell on this, and DataInventory definitely does. I'll do this.